static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -408,12 +414,13 @@ function WidgetBuilder({
   function handleDisplayTypeOrTitleChange<
     F extends keyof Pick<State, 'displayType' | 'title'>
   >(field: F, value: State[F]) {
-    trackAdvancedAnalyticsEvent('dashboards_views.add_widget_in_builder.change', {
+    trackAdvancedAnalyticsEvent('dashboards_views.widget_builder.change', {
```

Comment:
What kind of information do we hope to get from tracking title change?

Reply:
I guess it could give us an idea of the flow of user interactions. New widget > title changed...

Reply:
this code was based on the analytic code present in the modal and thought it could be useful here too